src/main/java/Problem1.java

```diff
@@ -1,8 +1,23 @@
 public class Problem1 {
 
     // Do not change signature (function name, parameters)
+
     public static int binarySearch(int[] data, int target) {
         // Add your solution
-        return -1; // Place holder
+        int index = -1; // index is going to be what im returning as the index
+        for (int i = 0; i < data.length; i++) // loop to go through array and
+        // find target
+        {
+            if (data[i] == target) // when the loop finds the index thats
+            // equal to the target, itll set index as i
+            {
+                index = i;
+                break;
+            } else {
+                index = -1;
```

Comment:
and even for linear search, this else case is not needed right? index will always be -1 from definition on line 7 unless it's changed on line 14.